cmd/main.go

```diff
@@ -73,6 +73,8 @@ var (
 	scheme                = runtime.NewScheme()
 	setupLog              = ctrl.Log.WithName("setup")
 	defaultNamespaceRegex = regexp.MustCompile("^octopus-(agent|worker)-.*")
+	// Set by linker flags at build time
+	version = "dev"
```

Comment:
Nit: should we consider changing this to be v0.0.0? Keeps it in line with the standard v-prepended versioning, and allows us to use semver parsing still.